files/project_team/IFCTunnelDep2.2.ifc

```diff
@@ -0,0 +1,624 @@
+#554= IFCUNITASSIGNMENT((#555,#556,#557,#558,#559));
+#555= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#556= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#557= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#558= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#559= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+/* IFCPROJECT aggregates IFCALIGNMENT */
+/* #560= IFCRELAGGREGATES('0Mym48ehwyHTreSA_asycx',#10,$,$,#553,(#4)); */
+#560= IFCRELAGGREGATES('0Mym48ehwyHTreSA_asycx',#10,$,$,#553,(#28));
+#561= IFCDIRECTION((0.0,1.0));
+#562= IFCPROJECTEDCRS('EPSG:3010',$,'SWEREF99','EPSG:5613',$,$,$);
+#563= IFCMAPCONVERSION(#41,#562,0.0,0.0,0.0,1.0,0.0,1.0);
+#564= IFCBUILTSYSTEM('3t_WSZJILAihHYV7qj$OaM',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
+#565= IFCRELREFERENCEDINSPATIALSTRUCTURE('12IPmDB715g8c8v_BgsfpL',#10,$,$,(#564),#14);
+#566= IFCRELDECLARES('3ZbEOw1KnBghKwwCyYrzKQ',#10,$,$,#553,(#564,#571,#580,#586));
+#567= IFCCLASSIFICATION('Triona AB',$,$,'My classification',$,$,$);
+#568= IFCRELASSOCIATESCLASSIFICATION('0YutlzF$H1shHmD0QN6uYf',#10,$,$,(#553),#567);
+#569= IFCDOCUMENTINFORMATION('ISBN: 978-91-7725-946-6','Titel: Krav - VGU, V\X2\00E4\X0\gars och gators utformning',$,'http://trafikverket.diva-portal.org/smash/get/diva2:1621114/FULLTEXT02.pdf',$,$,$,$,$,(#570),$,$,'application/pdf',$,$,$,$);
+#570= IFCORGANIZATION('http://www.trafikverket.se','Trafikverket',$,$,$);
+#571= IFCACTOR('117jPaib56iOR8Rdl1Yi2$',#10,$,$,$,#570);
+#572= IFCRELASSOCIATESDOCUMENT('1WYZaM$Cn0IAmODa0GojpI',#10,$,$,(#553),#569);
+#573= IFCDATASETINFORMATION('Dataset-xxx','Dataset-xxx-xml',$,'http://xxx.yyy.se/Datasets/Dataset-xxx.xml',$,$,$,$,$,$,$,$,'text/xml',$,$,$,$,'http://geosciml.org/geosciml/2.0/xsd/geosciml.xsd');
+#574= IFCRELASSOCIATESDATASET('161HpH4mjEKB8XVgoqkATr',#10,$,$,(#553),#573);
+#575= IFCLIBRARYINFORMATION('My library','1.0',#578,'2023-03-14T08:45:11','http://xxx.yyy.se/Libraries/MyLib.ifc',$);
+#576= IFCPERSON($,'Wikstr\X2\00F6\X0\m','Lars',$,$,$,$,$);
+#577= IFCORGANIZATION($,'Triona AB',$,$,$);
+#578= IFCPERSONANDORGANIZATION(#576,#577,$);
+#579= IFCRELASSOCIATESLIBRARY('3Va2BezgP7UweF_ghkVeRx',#10,$,$,(#553),#575);
+#580= IFCARCHELEMENTTYPE('3BEP_AU0P0qvjMvCQ6PQ4s',#10,'Arch-123',$,$,$,$,$,'MyArchType',.USERDEFINED.);
+#582= IFCARCHELEMENT('2S6rF0Ns966RXaJ930uR0a',#10,$,$,$,$,$,$,$);
+#583= IFCRELCONTAINEDINSPATIALSTRUCTURE('0t33FEZaGa4hTN7NqXrtVJ',#10,$,' Container for Elements',(#582),#22);
+#584= IFCRELDEFINESBYTYPE('3FJPWOudLCogmKkxND_ZFm',#10,$,$,(#582),#580);
+#585= IFCRELASSIGNSTOGROUP('10zRC93jfDv9vL0oh5cY3Z',#10,$,$,(#582),$,#564);
+#586= IFCPROPERTYSETTEMPLATE('2e3pgpwerBwuLkbm22Ui$g',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#587));
+#587= IFCSIMPLEPROPERTYTEMPLATE('2PpeGNxWn0EPRqQfXiVCKo',#10,'DrillingDiameter',$,.P_SINGLEVALUE.,'IfcPositiveLengthMeasure',$,$,$,$,$,$);
+/* IFCSITE (#4) positioned relative to start referent (#478) */
+#1000= IFCRELPOSITIONS('',#10,$,$,#478,(#4));
```

Comment:
Guid is empty. It feels weird that the checker does not complain. Have to look into that.